pkg/util/listblocks/listblocks.go

```diff
@@ -60,18 +65,18 @@ func LoadMetaFilesAndDeletionMarkers(ctx context.Context, bkt objstore.BucketRea
 	})
 
 	if err != nil {
-		return nil, nil, err
+		return nil, nil, nil, err
 	}
 
 	if showDeleted {
 		deletionTimes, err = fetchDeletionTimes(ctx, bkt, deletionMarkerFiles)
 		if err != nil {
-			return nil, nil, err
+			return nil, nil, nil, err
 		}
 	}
 
 	metas, err = fetchMetas(ctx, bkt, metaPaths)
-	return metas, deletionTimes, err
+	return metas, deletionTimes, noCompactBlocks, err
```

Comment:
It would be nice to fetch "reasons" and timestamps from no-compact markers, and return those.